apps/web/playwright/bookings-list.e2e.ts

```diff
@@ -111,3 +287,145 @@ async function createBooking({
+      slug: slugify(`bookings-test-team-2-${randomString(5)}`),
+    },
+  });
+
+  const teamTwo = await prisma.team.create({
+    data: {
+      name: "bookings-test-team-2",
+      slug: slugify(`bookings-test-team-2-${randomString(5)}`),
+    },
+  });
+
+  // create memberships
+  await prisma.membership.createMany({
+    data: [
+      {
+        userId: owner1.id,
+        teamId: teamOne.id,
+        accepted: true,
+        role: "OWNER",
+      },
+      {
+        userId: commonUser.id,
+        teamId: teamOne.id,
+        accepted: true,
+        role: "MEMBER",
+      },
+      {
+        userId: team1_teammate1.id,
+        teamId: teamOne.id,
+        accepted: true,
+        role: "MEMBER",
+      },
+      {
+        userId: owner2.id,
+        teamId: teamTwo.id,
+        accepted: true,
+        role: "OWNER",
+      },
+      {
+        userId: commonUser.id,
+        teamId: teamTwo.id,
+        accepted: true,
+        role: "MEMBER",
+      },
+      {
+        userId: team2_teammate1.id,
+        teamId: teamTwo.id,
+        accepted: true,
+        role: "MEMBER",
+      },
+    ],
+  });
+  return {
+    owner1,
+    owner2,
+    commonUser,
+    team1_teammate1,
+    team2_teammate1,
+    teamOne,
+    teamTwo,
+  };
+};
+
+const bookEvent = async ({
+  pageFixture,
+  team,
+  eventType,
+  user,
+}: {
+  pageFixture: Fixtures["page"];
+  team?: {
+    id: number;
+    slug: string | null;
+    name: string;
+  };
+  user?: {
+    id: number;
+    name: string | null;
+    username: string | null;
+  };
+  eventType: {
+    id: number;
+    title: string;
+    slug: string;
+  };
+}) => {
+  if (team?.slug) {
+    await pageFixture.goto(`/team/${team.slug}/${eventType.slug}/`);
+  } else if (user?.username) {
+    await pageFixture.goto(`/${user.username}/${eventType.slug}/`);
+  }
+  await selectFirstAvailableTimeSlotNextMonth(pageFixture);
+  await bookTimeSlot(pageFixture);
+  await expect(pageFixture.locator("[data-testid=success-page]")).toBeVisible();
+
+  // The title of the booking
+  let BookingTitle = "";
+  if (team?.name) {
+    BookingTitle = `${eventType.title} between ${team.name} and ${testName}`;
+  } else if (user?.name) {
+    BookingTitle = `${eventType.title} between ${user.name} and ${testName}`;
+  }
+  await expect(pageFixture.locator("[data-testid=booking-title]")).toHaveText(BookingTitle);
+  // The booker should be in the attendee list
+  await expect(pageFixture.locator(`[data-testid="attendee-name-${testName}"]`)).toHaveText(testName);
+};
+
+const assertBookingVisibleFor = async ({
+  user,
+  pageFixture,
+  eventType,
+  shouldBeVisible,
+}: {
+  user: { apiLogin: () => Promise<APIResponse> };
+  pageFixture: Fixtures["page"];
+  eventType: { title: string };
+  shouldBeVisible: boolean;
+}) => {
+  await user.apiLogin();
+  await pageFixture.goto(`/bookings/upcoming`);
+  const upcomingBookings = pageFixture.locator('[data-testid="upcoming-bookings"]');
```

Comment:
I see you use locator `data-testid="upcoming-bookings"` which cover bookings:

<img width="1509" alt="Screenshot 2024-02-15 at 13 46 43" src="https://github.com/calcom/cal.com/assets/42170848/09c188f3-0f61-4753-8111-be8c1eb26432">

but it ignores today's bookings with locator "today-bookings":
<img width="1506" alt="Screenshot 2024-02-15 at 13 47 04" src="https://github.com/calcom/cal.com/assets/42170848/6790f0f0-c349-4fd3-a397-c5e014e1c523">

so the `assertBookingVisibleFor` should be adjusted, because it is ignoring today's meetings and in case a test has a meeting today the test would fail even if booking is visibile.

Reply:
[commit](https://github.com/calcom/cal.com/pull/13591/commits/844106bc5958d6e33072e6b66f646b7235070ee3)

not sure if this would fix this issue,
tested on my end, all the tests pass!!

Reply:
Amazing! Yeah to the best of my understanding that should work! 🙏